package.json

```diff
@@ -9,7 +9,7 @@
     "build": "lerna run --parallel --scope \"@mui/*\" build",
     "build:codesandbox": "NODE_OPTIONS=\"–max_old_space_size=4096\" lerna run --concurrency 8 --scope \"@mui/*\" build",
     "release:version": "lerna version --no-changelog --no-push --no-git-tag-version --force-publish=@mui/core-downloads-tracker",
-    "release:build": "lerna run --parallel --scope \"@mui/*\" build --skip-nx-cache",
+    "release:build": "lerna run --concurrency 8 --scope \"@mui/*\" build --skip-nx-cache",
```

Comment:
Would this be a problem in the future, especially with the hard-coded `8`? cc @michaldudak in case you know what's the best way to fix this problem.

Reply:
This actually fixes the problem with memory usage. We have other cases like this already in other scripts.

Reply:
Could you approve this PR if this looks right. Only after adding this change did the CI pass.

Reply:
`--parallel` is obsolete https://lerna.js.org/docs/lerna6-obsolete-options#obsolete-options